lambda/authz-js/index.ts

```
@@ -0,0 +1,57 @@
+  if (!authHeader || !authHeader.startsWith("Bearer ")) {
+    return unauthorized("Missing or malformed Authorization header");
+  }
+
+  const token = authHeader.substring(7); // Remove 'Bearer ' prefix
+
+  try {
+    await verifier.verify(token);
+    return request;
+  } catch (err) {
+    console.error("JWT verification failed:", err);
+    return unauthorized("Invalid or expired token: " + err);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)